src/desktop/CMakeLists.txt

```diff
@@ -363,6 +363,8 @@ else()
 	# Android can neither deal with exporting multiple files nor can it spawn an
```

Comment:
Outdated/incomplete comment now

Reply:
Updated the comments. The main window one got roped into it too because it's all just consolidated now.